py4j-web/requirements-doc.txt

```diff
@@ -1 +1 @@
-Sphinx~=1.4
+Sphinx
```

Comment:
I recently started to use pip-tools to pin our dependencies. Without going to such length, should we at least require a specific major versions and update if needed, e.g.,

```
Sphinx >=4.4,<5.0
```

Reply:
Sounds good!